src/Microsoft.AspNetCore.ResponseCaching/ResponseCache/IResponseCacheEntry.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.ResponseCaching
+{
+    public interface IResponseCacheEntry
+    {
+        string Key { get; set; }
```

Comment:
These keys can be long. We might choose to store by a hash of it inside the cache. I'll leave it in for now with a TODO to remove it if not needed.